spec/helpers/govuk_link_helper_spec.rb

```diff
@@ -345,20 +366,40 @@ def url_for(path)
       end
     end
 
-    context "when new_tab: true" do
-      let(:kwargs) { { new_tab: true } }
+    describe "new_tab" do
```

Comment:
Is this a duplicate of the above, or is this for a different helper? (unclear from the diff)

Reply:
Yeah, this is for `govuk_button_link_to`. At some point I'll reorganise these into shared contexts.

Reply:
Ah cool. Wasn’t obvious from the diff. Moving to separate files might be enough. I’m never convinced by shared contexts as it’s hard to work out where they’re defined.

Reply:
Yeah that's the issue I have with them too, I want something I can jump to in my editor but the string names don't help.